_haikus/lobster.md

```diff
@@ -6,5 +6,5 @@ test: false
 ---
 
 Meandering thoughts<br>
-shall a spiny, smol lobster<br>
+shall a spiny, small lobster<br>
```

Comment:
I'm aware that [smol](https://www.urbandictionary.com/define.php?term=Smol) is slang. If that's intended, then this should be dropped.